python/nav/models/arnold.py

```diff
@@ -45,10 +45,16 @@ class Identity(models.Model):
     mac = models.CharField(db_column='mac', max_length=17)
     status = VarcharField(db_column='blocked_status', choices=STATUSES)
     justification = models.ForeignKey(
-        'Justification', on_delete=models.CASCADE, db_column='blocked_reasonid'
+        'Justification',
+        on_delete=models.CASCADE,
+        db_column='blocked_reasonid',
+        related_name="identities",
     )
     interface = models.ForeignKey(
-        Interface, on_delete=models.CASCADE, db_column='swportid'
+        Interface,
+        on_delete=models.CASCADE,
+        db_column='swportid',
+        related_name="identities",
```

Comment:
My only nitpick here seems to be that the term "identities" is pretty vague outside the confines of the Arnold ecosystem (the default `identity_set` was no better).

I would argue that related names on models outside of the Arnold namespace should be be more qualified, something like `arnold_identities`.